app/experimenter/legacy-ui/rapid/components/experiments/ResultsTable.tsx

```diff
@@ -0,0 +1,270 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+
+import {
+  SIGNIFICANCE_TIPS,
+  RESULT_COLUMN_TIPS,
+} from "experimenter-rapid/components/experiments/tooltips";
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import { AnalysisPoint, ExperimentData } from "experimenter-types/experiment";
+
+enum Significance {
+  POSITIVE,
+  NEGATIVE,
+  NEUTRAL,
+}
+
+// A mapping of metric label to relevant statistic. This is
+// used to see which statistic will be used for each metric.
+const resultsMetricsMap = {
+  retained: "binomial",
+  search_count: "mean",
+  identity: "count",
+};
+
+const setPopulationPercentages = (results, totalPopulation) => {
+  Object.keys(results).forEach((branch) => {
+    const populationCount = results[branch]["identity"]["point"];
+    const populationPercent = Math.round(
+      (populationCount / totalPopulation) * 100,
+    );
+    results[branch]["identity"]["percent"] = populationPercent;
+  });
+};
+
+const getFeatureNameMappings = () => {
+  // Create feature-name mappings.
+  const featureNameMappings = {};
+  featureOptions.forEach((featureRow) => {
+    featureNameMappings[featureRow["value"]] = featureRow["name"];
+  });
+  return featureNameMappings;
+};
+
+const getResultMetrics = (featureData) => {
+  // This is used in conjunction with resultsData as an ordered
+  // list of metrics to display in the table from left to right
+  const resultsMetricsList = [
+    {
+      value: "retained",
```

Comment:
These strings are littered in a few places, 

- `"retained"`
- `"search_count"`
- `"identity"`

Might be worth pulling those into constants somewhere?